plugins/Sid/CMakeLists.txt

```diff
@@ -0,0 +1,51 @@
+INCLUDE(BuildPlugin)
+
+INCLUDE_DIRECTORIES(resid)
```

Comment:
If you parse `resid/CMakeLists.txt` below, why did you include that here?

Reply:
I don't understand the question.  Can you please rephrase?

Reply:
Are you confusing `INCLUDE_DIRECTORY` with `ADD_SUBDIRECTORY` perhaps?

Reply:
Oh, yes. My mistake.